pkg/netutil/netutil_windows.go

```diff
@@ -0,0 +1,58 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package netutil
+
+const (
+	DefaultNetworkName = "nat"
+	DefaultID          = 0
+	DefaultCIDR        = "10.1.0.0/24"
```

Comment:
Why is this different from Linux?

Reply:
I don't think it needs to be, I was using the same values that I had manually configured to test